gateway/changelog/0.19.0.md

```diff
@@ -0,0 +1,11 @@
+# Features
+
+- New query planner, it is still work in progress, but some failing queries now work.
+
+# Changes
+
+- Composition now validates that subgraphs do not define `@join__Graph`.
```

Comment:
Did this impact the gateway?  Sounds like it wouldn't, but I may be misunderstanding

Reply:
oh right, gateway changelog